ouroboros-consensus/docs/Testing.md

```diff
@@ -307,6 +307,19 @@ far behind the wallclock, we might not have sufficient knowledge to translate
 the wallclock to a `SlotNo`, although we _can_ always translate the `SlotNo`
 at the tip of the chain to a `UTCTime`.
 
+
+## Golden tests
+
+Golden tests are specific to each era:
+
+- [`Test.Consensus.Byron.Golden`](ouroboros-consensus-byron-test/test/Test/Consensus/Byron/Golden.hs)
+- [`Test.Consensus.Cardano.Golden`](ouroboros-consensus-cardano-test/test/Test/Consensus/Cardano/Golden.hs)
+- [`Test.Consensus.Shelley.Golden`](ouroboros-consensus-shelley-test/test/Test/Consensus/Shelley/Golden.hs)
```

Comment:
When I open the [rendered version of this file](https://github.com/input-output-hk/ouroboros-network/blob/2f78fc4a7ad8cad80af0fa6dacbed7234e98d228/ouroboros-consensus/docs/Testing.md), the links do not work. I think they need to be absolute:

```suggestion
- [`Test.Consensus.Byron.Golden`](/ouroboros-consensus-byron-test/test/Test/Consensus/Byron/Golden.hs)
- [`Test.Consensus.Cardano.Golden`](/ouroboros-consensus-cardano-test/test/Test/Consensus/Cardano/Golden.hs)
- [`Test.Consensus.Shelley.Golden`](/ouroboros-consensus-shelley-test/test/Test/Consensus/Shelley/Golden.hs)
```

Reply:
Ups. I forgot to check this myself.